
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

<title>SpaceTx: experimental and computational methods compartison for spatially resolved transcriptomics</title>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/style.css" />
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Marker Gene Selection" href="1.marker-gene-selection.html" />
    <link rel="prev" title="Topics" href="../analysis_topics.html" /> 
  </head><body>
  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
    <div class="container-fluid">
      <a class="navbar-brand" href="../index.html">SpaceTx</a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
          <li><a class="nav-link" href="../topics.html">Topics</a></li>
          <li><a class="nav-link" href="../examples.html">Examples and Code</a></li>
          <li><a class="nav-link" href="../contribute.html">Contributing Guidelines</a></li>
        </ul>
        <form class="d-flex" action="../search.html" method="get">
          <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" name="q">
          <button class="btn btn-outline-success" type="submit">Search</button>
        </form>
      </div>
    </div>
  </nav>

    <div id="wrapper" class="container-fluid">
      <div id="main" class="row">
        <div id="content" class="col-md-9">
          <div class="container-fluid">
            
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="cell-type-clustering">
<h1>Cell Type Clustering<a class="headerlink" href="#cell-type-clustering" title="Permalink to this headline">¶</a></h1>
<p>Consensus clustering</p>
<p>Computational strategies:</p>
<ul class="simple">
<li><p>(Eeshit DV) SIMLR (Single-cell Interpretation via Multi-kernel LeaRning) learns an appropriate distance metric for clustering and accounts for drop-outs.  Can also be run as a consensus clustering strategy. (<a class="reference external" href="https://github.com/BatzoglouLabSU/SIMLR">https://github.com/BatzoglouLabSU/SIMLR</a>).</p></li>
<li><p>(Peter K) PAGODA (Pathway And Gene set Over-Dispersion Analysis) resolves multiple aspects of gene expression heterogeneity by testing gene sets for coordinated variability among measured cells                                                                             (<a class="reference external" href="http://hms-dbmi.github.io/scde/pagoda.html">http://hms-dbmi.github.io/scde/pagoda.html</a>).</p></li>
<li><p>(Zizhen Y) scrattch.hicat (Hierarchical, Iterative Clustering for Analysis of Transcriptomics) iteratively repeats a multi-step approach on each cluster using different variable genes until no new clusters can be defined, then bootstraps the iterative process (<a class="reference external" href="https://github.com/AllenInstitute/scrattch.hicat">https://github.com/AllenInstitute/scrattch.hicat</a>).</p></li>
<li><p>(Trygve B) - Precursor pipeline used for human MTG</p></li>
<li><p>(Kenneth H) proMMT (Probabilistic Mixture Modeling for Transcriptomics) uses an EM algorithm to iteratively define clusters using the Bayesian information criterion and identify a sparse set of genes varying across clusters (Harris et al 2018; “Classes and continua…”; PLoS Biol.).</p></li>
</ul>
<p>Most methods were run multiple times with different parameter settings.</p>
<p>Cluster consensus:</p>
<ul class="simple">
<li><p>(Trygve B) - Cluster comparisons using Adjusted Rand, VI, etc. and showing which clusters subclasses /types are split more by various methods</p></li>
<li><p>(Brian A) - Consensus mouse and human clusters were defined using clusterExperiment R library, which clusters the co-occurrence matrix (<a class="reference external" href="https://github.com/epurdom/clusterExperiment">https://github.com/epurdom/clusterExperiment</a>).</p></li>
</ul>
<p>Mouse clusters were defined on VISp + ALM data set and 68 ALM-specific clusters were removed from the consensus clustering, for 191 total clusters</p>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
          </div>
        </div>
        <div id="sidebar" class="col-md-3">
          <div class="container-fluid">
            <h3>Navigation</h3>
            <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../results.html">Contributing Guidelines</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../analysis_topics.html">Topics</a><ul class="current">
<li class="toctree-l2 current"><a class="current reference internal" href="#">Cell Type Clustering</a></li>
<li class="toctree-l2"><a class="reference internal" href="1.marker-gene-selection.html">Marker Gene Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="2.experimental-methods.html">Experimental Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="3.cell-type-mapping-and-consensus.html">Cell Type Mapping and Combined Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="4.data-visualization.html">Data Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../discussion.html">Topics</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../discussion/discussion.html">Discussion</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../examples.html">Examples and Code</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../examples/0.cell-type-mapping-methods.html">Cell Type Mapping Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../examples/1.combinded-mapping-results.html">Combined mapping results</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../contribute.html">Contributing Guidelines</a></li>
</ul>

          </div>
        </div>
      </div>
    </div>

  <div id="footer">
    <p>
      © 2021 SpaceTx consortium<br>
    </p>
  </div>

  </body>
</html>