
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

<title>SpaceTx: experimental and computational methods compartison for spatially resolved transcriptomics</title>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/style.css" />
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Data Visualization" href="6.data-visualization.html" />
    <link rel="prev" title="Data Formatting Standards and Computational Pipelines" href="4.data-formatting-pipelines.html" /> 
  </head><body>
  <nav id="main-navbar" class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
    <div class="container">
      <a class="navbar-brand" href="../index.html"><img id="main-logo" src="/_static/spacetx_logo_web.svg"></a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
          <li><a class="nav-link" href="../people.html">People</a></li>
          <li><a class="nav-link" href="../data.html">Data</a></li>
          <li><a class="nav-link" href="../spacejam.html">SpaceJam Hackathon</a></li>
          <li><a class="nav-link" href="../examples.html">Examples and Code</a></li>
          <li><a class="nav-link" href="../contribute.html">Contributing Guidelines</a></li>
        </ul>
        <form class="d-flex" action="../search.html" method="get">
          <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" name="q">
          <button class="btn btn-success" type="submit">Search</button>
        </form>
      </div>
    </div>
  </nav>
    <div class="spacer"></div>

    <div id="wrapper">
      <div id="main" class="container">
        <div id="content">
          
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="cell-type-mapping-and-combined-results">
<span id="cell-type-mapping"></span><h1>Cell Type Mapping and Combined Results<a class="headerlink" href="#cell-type-mapping-and-combined-results" title="Permalink to this headline">¶</a></h1>
<ul class="simple">
<li><p><strong>Cell Type Mapping</strong></p></li>
</ul>
<p>This is the first time spatial transcriptomics data has been analyzed and compared across methods for cell type determination. We developed approaches to combine multiple cell type mapping methods and applied them to data sets from five experimental methods.</p>
<p>A primary output of segmentation in each of the image-based experimental methods is a table of putative cells which each have a count of the number of molecules per gene as well as a soma location.  In this section we present results for MERFISH, but can do the same for other methods.  Cells located outside of the primary visual cortex (VISp) based on expert annotation or that do not pass quality control filters are excluded from further analysis.</p>
<blockquote>
<div><p>We next aimed to assign a cell type and associated confidence to each putative cell based on gene molecule counts.  This was done using a combined analysis approach, where the strategies below were used to map cells to cell types from the taxonomy defined above, and the results of these strategies were then combined to produce a single combined cell type call with associated confidence score for each remaining cell.</p>
</div></blockquote>
<ul>
<li><p>Annotation of VISp area for mapping
Cell locations were visualized in Napari <a class="reference external" href="https://napari.org">https://napari.org</a> and manually
annotated to contain the largest possible area of primary visual cortex.
Layers were manually annotated based on cell density and gene expression.</p></li>
<li><p>Cell Mapping strategies (including cell QC/filtering steps)</p>
<ul class="simple">
<li><p>QC criteria</p></li>
<li><p>Baysor (Viktor Petukhov)  <a class="reference external" href="https://github.com/kharchenkolab/Baysor">Github repo</a>, <a class="reference external" href="https://www.nature.com/articles/s41587-021-01044-w">Nature Biotechnology (2021)</a> or <a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.10.05.326777v1">BioRxiv preprint</a></p></li>
<li><p>mFISHtools (Jeremy Miller) <a class="reference external" href="https://github.com/AllenInstitute/mfishtools/">Github repo</a></p></li>
<li><p>Eeshit’s method [Eeshit]</p></li>
<li><p>Renee’s method: Mapping spatially-defined cells to canonical cell types from scRNA-seq using FR-Match</p></li>
<li><p>pciSeq <a class="reference external" href="https://github.com/acycliq/pciSeq">Github repo</a></p></li>
<li><p>Yilin’s method [Yilin]</p></li>
</ul>
</li>
<li><p><strong>Combination of mapping results</strong>
- negative weighting
- geometric median</p></li>
<li><p><strong>Visium Data Single Cell Mapping</strong></p>
<ul class="simple">
<li><p>Visium is often described as unbiased and transcriptome-wide, since it’s not based on a priori selection of genes to be examined and targets all poly-adenlated transcripts. However, the current Visium platform does not provide single cell resolution and, though dependent on tissue, the common estimate is that approximately 1-10 cells contribute to the observed gene expression at each spot. The “mini-bulk” property of spots means that spots rarely can be assigned to a single cell type, and more common is to characterize these w.r.t. cell type compositions (e.g., proportion of cell types present at each location). For such purposes, several methods where single cell/nuclei data is used as a guide reference have been proposed, with the shared objective of decomposing the joint expression profiles observed in a spot into contributions from the cell types defined in the single cell/nuclei data.</p></li>
<li><p><strong>stereoscope</strong> and <strong>Tangram</strong> were used to deconvolve the spatial Visium data using non-spatial single cell data, a procedure we refer to as “mapping” of the latter onto the former. In short, once presented with the Visium data, both these approaches will provide a proportion estimate of each cell type within every spot, thus allowing the spatial distribution of each cell type to be charted. Although the final products are similar, the two methods differ substantially in design; stereoscope relying on a probabilistic framework where both data modalities are modelled with a negative binomial distribution, while Tangram cast the the task more as a optimization problem where a map between single cells and spatial locations.</p></li>
<li><p>For consistency, the cell type mapping analysis focuses on the small region that constitutes the VISp as shown in Figure 1; even though spatial gene expression data was generated from a larger part of the tissues (about half of a coronal mouse brain section fit onto the capture area). We aim to evaluate the mapping from each method, as well as comparing these two to each other. Nevertheless, establishing a ground truth for the proportion values is arguably even harder than for the cell type identities in the image-based methods, hence concurrence with well-known locations of cell types (e.g., Layer associated) will serve as the basis of our evaluation.</p></li>
</ul>
<a class="reference internal image-reference" href="../_images/visium.png"><img alt="../_images/visium.png" src="../_images/visium.png" style="width: 800px;" /></a>
<p><strong>Figure 1</strong> | A) zoom in on the regions of interests. Spots included in the analysis are marked on the tissue with black circles. The tissue edged to which distance is measured is indicated by a dashed red line. B) Pearson correlation values between cell type proportion estimates from stereoscope and Tangram. The star (*) on Meis2 indicates that this correlation did not have a significant p-value. C) Smoothed curves (loess smoothing) of the cell type proportions when plotted as a function of distance to the tissue edge (red in A).</p>
<ul class="simple">
<li><p>By computing the correlation (Pearsons’s r) between proportion estimates for each cell type it’s possible to quantitatively assess how results from the two methods relate. A  significant positive correlation between the proportion estimates could be observed for all cell types except Meis2, where the correlation was negative but also non-significant (p = 0.36), see Figure 1B.  High correlation values were observed for several of the layer types as well as Macrophages and Astrocytes. Most of the cell types with poor correlation were - according to the proportion estimates - lowly abundant in the tissue, implying mapping of rare cell types likely are more challenging to map and the result associated with higher uncertainty.</p></li>
<li><p>Next, we were interested in how the different layer cell types were distributed along the axis orthogonal to the tissue edge, i.e., when travelling further into the tissue (blue arrow Figure 1A). We thus measured the shortest distance for every spot to the tissue edge and modelled the cell type proportion values as a function of this distance, loess (locally estimated scatterplot smoothing) curve smoothing was used to get a more continuous graph, and to better capture the general trends in the data, see Methods. The two methods by en large agreed; albeit not always unimodal, the cell type distributions had one dominant major mode - overlapping well across methods - and exhibited the expected right shift trend (layer types with higher numbers being more prevalent deeper into the cortex and vice versa), see Figure 1C. For some cell types, the distributions were multimodal indicating potential “mismapping”, still in the cases where both methods independently located these peaks at almost identical positions the explanation might be biological or experimental.</p></li>
</ul>
</li>
</ul>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
        </div>
      </div>
    </div>

  <div id="footer">
    <div class="container">
      <ul class="current">
<li class="toctree-l1 current"><a class="reference internal" href="../analysis_topics.html">Topics</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="0.cell-type-clustering.html">Cell Type Identification in single cell RNASeq data</a></li>
<li class="toctree-l2"><a class="reference internal" href="1.marker-gene-selection.html">Marker Gene Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="2.experimental-methods.html">Experimental Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="3.segmentation.html">Cell Segmentation</a></li>
<li class="toctree-l2"><a class="reference internal" href="4.data-formatting-pipelines.html">Data Formatting Standards and Computational Pipelines</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">Cell Type Mapping and Combined Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="6.data-visualization.html">Data Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../examples.html">Examples and Code</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../examples/0.cell-type-mapping-methods.html">Cell Type Mapping Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../examples/1.combinded-mapping-results.html">Combined Mapping Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="../examples/2.consensus_msmfish.html">SSAM Notebook: Multiplexed smFISH SSAM analysis</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../results.html">Data and Results</a></li>
<li class="toctree-l1"><a class="reference internal" href="../discussion.html">Discussion</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../discussion/about-spacetx.html">About SpaceTx</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/discussion.html">Introduction</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/related-projects.html">Related Projects and Ongoing Work</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/commercial-systems.html">Commercial Systems for Spatial Transcriptomics</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../contribute.html">Contributing Guidelines</a></li>
</ul>

      <p class="copyright">
        © 2022 SpaceTx consortium<br>
      </p>
    </div>
  </div>

  </body>
</html>